main.tf

```diff
@@ -280,10 +281,10 @@ resource "aws_iam_role_policy_attachment" "ecs_exec" {
 # Service
 ## Security Groups
 resource "aws_security_group" "ecs_service" {
-  count       = local.enabled && var.network_mode == "awsvpc" ? 1 : 0
+  count       = local.create_security_group ? 1 : 0
   vpc_id      = var.vpc_id
   name        = module.service_label.id
-  description = "Allow ALL egress from ECS service"
+  description = "Controls traffic to/from ECS service ${module.this.id}"
```

Comment:
I believe changing the description to a SG causes a recreate of the SG... That said, I believe this is a very poor description so I agree with your update. People will need to deal for the better of all I guess (including myself). At least we have the lifecycle rule which is a bonus.

Reply:
The description should be an input variable like this

https://github.com/cloudposse/terraform-aws-security-group/blob/e5bef85c74c4229a8e241fd78a565ef4ce5b9e25/exports/security-group-variables.tf#L22

We recently reverted security group rule changes and introducing another breaking change is not the best approach here.

Reply:
Ah, that makes sense. Changed to an input variable with the default set to the original description: 45c1ed5